Docs/CenterlineDisassembly.md

```diff
@@ -0,0 +1,21 @@
+# Centerline disassembly
+
+This module breaks down a bifurcated centerline model into parts.
+
+The centerline must have been created with the 'Extract centerline' module.
+
+![CenterlineDisassembly](CenterlineDisassembly_0.png)
+
+### Usage
+
+Select a centerline model node, the components to extract and apply. The result can be browsed in the 'Models' module's widget.
+
+The components of a centerline can be:
+
+      - bifurcation parts
+      - branches, centerline parts that exclude the bifurcations
+      - centerlines, i.e, one complete centerline from the first endpoint to every other endpoint.
+
+### Disclaimer
+
+Use at your own risks.
```

Comment:
This is true for the entire 3D Slicer application and all modules in all extensions. If you really want then you can remind people in the extension's documentation. So, I would only add a warning for a specific module if there is something unusual about this module or you have somet more specific information/advice than just "use at your own risks".